sdk/dotnet/Pulumi.Automation/Pulumi.Automation.csproj

```diff
@@ -0,0 +1,63 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Nullable>enable</Nullable>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Authors>Pulumi</Authors>
+    <Company>Pulumi Corp.</Company>
+    <Description>The Pulumi .NET SDK lets you write cloud programs in C#, F#, and VB.NET.</Description>
+    <PackageProjectUrl>https://www.pulumi.com</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/pulumi/pulumi</RepositoryUrl>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <PackageIcon>pulumi_logo_64x64.png</PackageIcon>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+  </PropertyGroup>
```

Comment:
I don't want to overcomplicate versioning. Ideally, we just version it in unison with the SDK. Preview label is fine, but this was the intention of keeping it in an `x` namespace originally. Do we need/want to maintain extra machinery of keeping this in a different assembly? 

While unconventional, we're doing the same thing in other languages and communicating risk of API churn to users via the announcements.

Reply:
I feel pretty strong about the separate assembly here. Versioning in line with Pulumi core is fine.

Reply:
While it's ultimately up to you guys, as a primarily dotnet dev I agree that the separate assembly is the way to go here.

Reply:
Here is my proposal: https://github.com/pulumi/pulumi/issues/6312
WDYT @orionstudt?

Reply:
I hadn't considered the difficulty of having a single solution that is only partially pre-release - but splitting it into a separate solution sounds like it would resolve that. Sounds good to me. How do you want me to handle that in this PR? Maybe just put an `<IsPackable>false` in the automation .csproj until you guys are ready to do that in a separate PR?

Reply:
I was going to suggest maybe just putting `<VersionSuffix>preview` in the automation .csproj until it is out of pre-release but I think the fact that you guys are calling `dotnet build dotnet.sln /p:VersionPrefix=${VERSION_PREFIX} /p:VersionSuffix=${VERSION_SUFFIX}` everytime would take priority over the the attribute in the .csproj.

Reply:
Disabling `IsPackable` in this PR makes sense to me. I'll follow-up with the solution split and a Makefile amendment.